ontology/catplus_ontology.ttl

```diff
@@ -352,6 +358,8 @@ cat:Observation a rdfs:Class, sh:NodeShape ;
     skos:definition "An observation is an abstract entity that captures the unit and value of a property into one node." ;
     sh:property [sh:path qudt:unit  ; sh:minCount 1 ; sh:maxCount 1] ;
     sh:property [sh:path qudt:value ; sh:minCount 1 ; sh:maxCount 1] ;
+    sh:property [sh:path cat:errorMargin ; sh:datatype xsd:decimal; 
+                sh:node [sh:property [sh:path qudt:unit ; sh:minCount 1 ], [sh:path qudt:value ; sh:minCount 1 ]]] ;

```

Comment:
question: what about having a `cat:ObservationWithErrorMargin` and a `cat:Observation` then you could say here that the error margin is again a `cat:Observation`. Not sure whether this is better or not, I just felt that this here `sh:node [sh:property [sh:path qudt:unit ; sh:minCount 1 ], [sh:path qudt:value ; sh:minCount 1 ]]` is again an Observation but one without Error Margin.